sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/error/JwtAuthException.java

```diff
@@ -18,6 +18,6 @@ public JwtAuthException(String message, Throwable cause) {
   }
 
   public JwtAuthException(Throwable cause) {
-    super(cause);
+    this(cause.getMessage(), cause);
```

Comment:
From my experience the message is often useless. E.g. the message for NullpointerException always is `null`.